.gitignore

```diff
@@ -17,3 +17,4 @@ addons/*
 
 .vscode
 .addons/*
+*.import
```

Comment:
Its a great idea to make sure that we're not pusing import files to the repository. Nice job!